vhdbuilder/packer/cleanup.sh

```diff
@@ -28,15 +28,18 @@ fi
 
 #clean up managed image
 if [[ -n "$AZURE_RESOURCE_GROUP_NAME" && -n "$IMAGE_NAME" ]]; then
-  if [[ "$MODE" != "default" ]]; then
+  if [[ "$MODE" != "default" && ${ARCHITECTURE,,} != "arm64" ]]; then
```

Comment:
`"$MODE" != "default"` should always resolve to true, thus it should be safe to remove entirely